expo-maps/src/Maps.tsx

```diff
@@ -17,13 +17,73 @@ const defaultNativeExpoMapViewProps: DefaultNativeExpoMapViewProps = {
 };
 
 export class ExpoMap extends React.Component<ExpoMapViewProps> {
+  private mapChildren() {
+    const childrenArray = React.Children.map(this.props.children, (child) => {
+      if (isNotSimpleType(child)) {
+        if (instanceOfExpoMarker(child)) {
```

Comment:
I'm not a JS expert, but I think you can obtain the type of the child doing something like this: `(child as React.ReactElement<any>).type`